pandas/core/indexers.py

```diff
@@ -208,7 +206,11 @@ def maybe_convert_indices(indices, n: int):
 
 def length_of_indexer(indexer, target=None) -> int:
     """
-    return the length of a single non-tuple indexer which could be a slice
+    Returns
```

Comment:
Docstrings ideally start with a one liner that says what the function does. Can you leave something like that?